libp2p.nimble

```diff
@@ -9,6 +9,7 @@ skipDirs      = @["tests", "examples", "Nim", "tools", "scripts", "docs"]
 
 requires "nim >= 1.2.0",
          "nimcrypto >= 0.4.1",
+         "https://github.com/Menduist/dnsclient.nim == 0.1.0",
```

Comment:
Should be probably moved to a status repo. 

Another point is that this doesn't help us determine the OS DNS or does it?

Reply:
I just forked it because I needed a few modifications.
But they just been [merged](https://github.com/ba0f3/dnsclient.nim/pull/5) upstream, so we can actually use the upstream repo.
Or we can fork it in status, as you want

Reply:
For the OS DNS, I parse `/etc/resolv.conf` on unix and use a function in windows (todo).
More details in my last comment